nautobot/extras/tables.py

```diff
@@ -915,6 +922,55 @@ class Meta(BaseTable.Meta):
         )
 
 
+class ObjectMetadataTable(BaseTable):
+    pk = ToggleColumn()
+    metadata_type = tables.Column(linkify=True)
+    assigned_object = tables.TemplateColumn(template_code=ASSIGNED_OBJECT, orderable=False)
+    actions = ButtonsColumn(
+        ObjectMetadata,
+        buttons=("delete"),
+    )
+    # This is needed so that render_value method below does not skip itself
+    # when metadata_type.data_type is TYPE_CONTACT_TEAM and we need it to display either contact or team
+    value = tables.Column(empty_values=[])
+
+    class Meta(BaseTable.Meta):
+        model = ObjectMetadata
+        fields = (
+            "pk",
+            "assigned_object",
+            "scoped_fields",
+            "value",
+            "metadata_type",
+            "actions",
+        )
+        default_columns = (
+            "pk",
+            "assigned_object",
+            "scoped_fields",
+            "value",
+            "metadata_type",
+            "actions",
+        )
+
+    def render_scoped_fields(self, record):
+        return render_json(record.scoped_fields, pretty_print=True)
+
+    def render_value(self, record):
```

Comment:
It would be great to include a test for each one of the method various return variants.

Reply:
These helper functions are just returning `render_json`/`render_markdown`/`render_boolean` which all have their respective rendering tests implemented already. I can build a test to validate if contact/team is rendered correctly.